packages/vite/src/utils/generator-utils.ts

```diff
@@ -347,6 +348,8 @@ export interface ViteConfigFileOptions {
   imports?: string[];
   plugins?: string[];
   coverageProvider?: 'v8' | 'istanbul' | 'custom';
+  setupFile?: string;
```

Comment:
I wonder if we really need to pass the `setupFile` here as the path is already hardcoded in some places. e.g. https://github.com/nrwl/nx/blob/6cc833013666bf62afde35871bb6d1871b39b886/packages/vite/src/generators/init/init.ts#L25

Reply:
i think this is safer and paves the way for better safety moving forward